### PR TITLE
chore(docs/tests): Add e2e tests description and requirements into co…

### PR DESCRIPTION
**Pull request by odockal**
…ntributing guide

### What does this PR do?
Includes information about e2e tests in contributing guide. 

### What issues does this PR fix or reference?
#2636

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Write tests! Please try to write some unit tests when submitting your PR.
 Run the tests using `yarn`:
```
Comment (benoitf) [inline CONTRIBUTING.md]: Run the unit and component tests using

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -305,10 +316,9 @@ Within Podman Desktop, we use the following frameworks and tools to build the de
 ### Testing
 
 Within Podman Desktop, we use the following for testing:
-
 - [Vitest](https://vitest.dev/): Unit tests - Written as `spec.ts` files.
 - [Testing Library](https://testing-library.com/): Component tests - Utilities and best practices for writing component tests.
-- [Playwright](https://playwright.dev/): Integration tests.
+- [Playwright](https://playwright.dev/): Integration and E2E tests.
```
Comment (benoitf) [inline CONTRIBUTING.md:322]:
```suggestion
- [Playwright](https://playwright.dev/): E2E tests.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -305,10 +316,9 @@ Within Podman Desktop, we use the following frameworks and tools to build the de
 ### Testing
 
 Within Podman Desktop, we use the following for testing:
-
```
Comment (benoitf) [inline CONTRIBUTING.md:308]: I suppose this blank line was added by prettier

Comment (odockal) [reply CONTRIBUTING.md:308]: Seems like it.

**Review by benoitf — APPROVED** [review]

**Pull request merged**